src/utils/__tests__/refs.jest.test.ts

```diff
@@ -0,0 +1,66 @@
+import { extractSourceFromRef, traverseObjUntilRef } from '..';
+
+describe('$ref utils', () => {
```

Comment:
These might be a good fit for the JSON package (where we already have a bunch of pointer helpers etc)?

Reply:
https://github.com/stoplightio/json/pull/38